src/components/calendar/Calendar.js

```diff
@@ -30,25 +30,26 @@ export default function Calendar() {
         
         fetchData()
     },[])
-    const eventAddHandler = (event) => {
-        saveEvent(event);
-        setEvents([...events, event]);
+    const eventAddHandler = async (event) => {
+        const savedEvent = await saveEvent(event);
+        setEvents([...events, savedEvent]);
```

Comment:
Do the current changes allow updates on an existing event?
If so, you will need to update this logic.